man/systemctl.xml

```diff
@@ -2134,6 +2134,19 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--timestamp</option></term>
```

Comment:
we usually suffix cmdline params that take values with `=` to indicate that. Hence please write `<option>--timestamp=</option>` here, and everywhere

Reply:
fixed